Exiled.API/Features/Player.cs

```diff
@@ -399,6 +399,16 @@ public RoleType Role
         /// </summary>
         public PlayerMovementState MoveState => ReferenceHub.animationController.MoveState;
 
+        /// <summary>
+        /// Gets the player's current animation.
+        /// </summary>
+        public int CurrentAnimation => ReferenceHub.animationController.curAnim;
+
+        /// <summary>
+        /// Gets a value indicating whether or not the player is sprinting.
+        /// </summary>
+        public bool IsSprinting => ReferenceHub.animationController.curAnim == 1;
```

Comment:
```suggestion
        public bool IsSprinting => CurrentAnimation == 1;
```

Also, change `IsJumping` like so.